config/README.rst

```diff
@@ -247,6 +264,12 @@ The corresponding section is named: ``[coverage]``.
 fail-under
   A minimal value of code coverage below which a test failure is issued.
 
+tox-ini-command
```

Comment:
I do not like the wording that much. It makes clear, that it is concerned with `tox.ini` but outside of the coverage section it can mean anything. 
```suggestion
tox-coverage-command
```

Reply:
I was thinking about moving the option to the `[tox]` section (as `coverage-command`) but I am not sure what's the best.

Reply:
I wold opt for the tox-section.

Reply:
Fixed.